app/collections/queries/getDraftCollections.ts

```diff
@@ -0,0 +1,47 @@
+import { resolver, Ctx, AuthorizationError } from "blitz"
+import db from "db"
+
+export default resolver.pipe(async ({ session }) => {
+  const draftCollections = await db.collection.findMany({
```

Comment:
Are we getting all of one's collection, not only the draft collections, in this query? (If so, shall we change to something like `getMyCollections`?)

Reply:
Fair point - will update!